pmm-app/src/pmm-qan/panel/components/Filters/Filters.test.tsx

```diff
@@ -1,11 +1,24 @@
-// Just a stub test
```

Comment:
Please remove this comments and add tests for filters

Reply:
For now, I just removed test file because anyway it was useless